app/src/main/java/ch/epfl/sweng/rps/ui/game/GameFragment.kt

```diff
@@ -75,7 +87,7 @@ class GameFragment : Fragment() {
                 binding.paperRBOpponent.isChecked = true
             }
             Hand.SCISSORS -> {
-                binding.scissorsRB.isChecked = true
+                binding.scissorsRBOpponent.isChecked = true
             }
             Hand.NONE -> throw IllegalStateException("Impossible")
```

Comment:
Why would we throw ? We decided together that Hand.NONE would represent the situation where the use times out whiole choosing a hand.

Reply:
You didn't want that idea. So generally, the current thing is event-based, not timeout-based. The choice of the button starts the process.

Reply:
Now I don't support time-based termination. I can implement it for the next sprint, and we could merge this one for event-based flow, meaning time == infinity